src/Transportation.cpp

```diff
@@ -0,0 +1,132 @@
+#include "../include/Transportation.h"
+
+// destructor
+Transportation::~Transportation()
+{
+    cout << "Transportation Object Destroyed!!!" << endl;
```

Comment:
What is the benefit ??

Reply:
This is a destructor; it can be used when the user wants to delete an object from a certain collection.